README.md

```diff
@@ -98,6 +98,22 @@ This project uses [CMake] and [Hunter] package manager.
    sudo make install
    ```
 
+#### OpenCL support on Linux
```

Comment:
Well, yes probably OpenGL will be installed automatically when you're installing OpenCL. But back to suggestion. As I said it's already in "Building from source" section where Hunter is enabled, so does it make sense to mention it again. Rename to "OpenCL support on Linux (with Hunter)"?